README.rst

```diff
@@ -74,4 +74,4 @@ Alternatively, you can use `Docker`_ and `Docker Compose`_ to install and run Sn
     $ docker build -t snappass .
     $ docker-compose up -d
 
-This will pull all dependencies, i.e. Redis and appropriate Python version (3.3), then start up snappass and Redis server. SnapPass server is accessible at: http://localhost:5000
+This will pull all dependencies, i.e. Redis and appropriate Python version (3.5), then start up snappass and Redis server. SnapPass server is accessible at: http://localhost:5000
```

Comment:
@nichochar This section of the README applies to Docker only, so we should only indicate the version of python on which the image is based - in this case, 3.5.